src/rail.h

```diff
@@ -341,6 +346,26 @@ static inline bool RailNoLevelCrossings(RailType rt)
 	return HasBit(GetRailTypeInfo(rt)->flags, RTF_NO_LEVEL_CROSSING);
 }
 
+/**
+ * Test if 90 degree turns are allowed between two railtypes.
+ * @param rt1 First railtype to test for.
+ * @param rt2 Second railtype to test for.
+ * @param def Default value to use if the rail type doesn't specify anything.
+ * @return True if 90 degree turns are allowed between the two rail types.
+ */
+static inline bool Rail90DegTurnDisallowed(RailType rt1, RailType rt2, bool def = _settings_game.pf.forbid_90_deg)
```

Comment:
Function name says "disallowed", but doc comment says it returns true if the turn is allowed. Negative bool variables/functions are confusing in general, try to make it positive.

Reply:
Well, comment is wrong, function is right.